### PR TITLE
CRM-15893 - PHP Notice fix

### PR DESCRIPTION
**Pull request by nganivet**

### CODE REVIEW

File: CRM/Contact/BAO/Group.php [hunk 1]
```diff
@@ -741,6 +741,7 @@ static public function getGroupListSelector(&$params) {
   static function getGroupList(&$params) {
     $config = CRM_Core_Config::singleton();
 
+    $select = $from = $where = $orderBy = $limit = '';
```
Comment (colemanw) [inline CRM/Contact/BAO/Group.php]: Doesn't this make line 760 redundant? Looks like that should be removed.

**Comment by nganivet** [comment]
Coleman - you are right, PR amended.

**Comment by colemanw** [comment]
@nganivet we do not do forward-merges from LTS to current versions, so if this change is also needed in 4.5 a pr would need to be submitted for that branch. Is that the case?

**Comment by nganivet** [comment]
@colemanw no PR needed for 4.5 since the function has been completely rewritten to use the API rather than a direct SQL query.

**Pull request merged**